packages/libwasmvm.nix

```diff
@@ -5,7 +5,10 @@
 }: let
   libwasmvmCommon = {
     pname = "libwasmvm";
-    nativeBuildInputs = with pkgs; [rust-bin.stable.latest.default];
+    nativeBuildInputs = with pkgs; [
+      (rust-bin.selectLatestNightlyWith (toolchain: toolchain.default))
+      # rust-bin.stable.latest.default
```

Comment:
I think nightly is 1.76 and stable is 1.73. We can go with either, just wanted to demonstrate how to use nightly

Reply:
Latest stable is 1.74. I would probably go with that over nightly as nightlife's are inherently unstable. Maybe leave the comment for future reference?